README.md

```diff
@@ -270,49 +353,58 @@ will automatically use chunked transfer encoding and send the respective header
 If you know the length of your body, you MAY specify it like this instead:
```

Comment:
Does this paragraph still apply? I see some room for improvement here with the updated API :+1: